pybamm/pybammdata.py

```diff
@@ -0,0 +1,48 @@
+import pooch
+
+
+class DataLoader:
+    def __init__(self):
```

Comment:
This class should include a docstring with Google-style docstring parameters to explain what it is about and a usage example or two. You should add the associated `.rst` files as well for it to render on the Sphinx documentation